libraries/AP_Math/SCurve.cpp

```diff
@@ -30,7 +30,7 @@ extern const AP_HAL::HAL &hal;
 
 #define SEG_INIT                0
 #define SEG_ACCEL_MAX           4
-#define SEG_TURN_IN             4
+#define SEG_TURN_IN             7
```

Comment:
This makes so much more sense to me now.  I was confused by this when it = 4.
Not a blocker for this PR as it is a nit pick but in a future PR I think it would be nice to remove the SEG_TURN_IN define and just use the SEG_ACCEL_END define.  We already have the time_turn_in() function so moving to the SEG_ACCEL_END define in that function makes the code easier to follow IMHO.

Reply:
I will do it for this PR I think. I was thinking the same thing as I looked at it.